src/pybind/mgr/cephadm/schedule.py

```diff
@@ -123,6 +124,35 @@ def validate(self) -> None:
                     f'Cannot place {self.spec.one_line_str()}: No matching '
                     f'hosts for label {self.spec.placement.label}')
 
+    def place_sidecars(
```

Comment:
in my view, sidecars are containers that are places within systemd units. If I see this correctly, we're talking about placing daemons next to other daemons, right?

Reply:
renamed